### PR TITLE
add zepto.fullpage@0.5.0 w/ git auto-update

### LINKED ISSUE #8342: [Request]  zepto.fullpage
**Library name:zepto.fullpage
**Git repository url:https://github.com/yanhaijing/zepto.fullpage.git
**npm package url(optional):** 
**License(s):MIT
**Official homepage:https://github.com/yanhaijing/zepto.fullpage
**Wanna say something? Leave message here:A zepto plugin to create fullpage scrolling webapps

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8342 
@x09326 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/yanhaijing/zepto.fullpage
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1223
   - Count of watchers: 113
   - Count of forks: 358
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8342, cc @yanhaijing

### CODE REVIEW

File: ajax/libs/zepto.fullpage/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "license": "MIT",
+  "dependencies": {
+    "zepto": "1.x"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/yanhaijing/zepto.fullpage.git",
+    "basePath": "src",
+    "files": [
+      "zepto.fullpage*"
```
Comment (PeterDaveHello) [inline ajax/libs/zepto.fullpage/package.json]: No `js.fullpage.js`?

Comment (kennynaoh) [inline ajax/libs/zepto.fullpage/package.json]: Yes, I think we need

**Comment by kennynaoh** [comment]
@x09326 
I have added `js.fullpage.js`.
Please help me review it again.
Thanks!

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
This branch need to rebase with the latest master branch.

**Comment by kennynaoh** [comment]
@x09326 
I have rebased to the latest master branch.
Please help me review it again.
Thanks!

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

File: ajax/libs/zepto.fullpage/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "license": "MIT",
+  "dependencies": {
+    "zepto": "1.x"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/yanhaijing/zepto.fullpage.git",
+    "basePath": "src",
+    "files": [
+      "+(zepto.fullpage|js.fullpage)*"
```
Comment (PeterDaveHello) [inline ajax/libs/zepto.fullpage/package.json]: I prefer the simplier usage `"*fullpage*"`?

**Comment by kennynaoh** [comment]
@x09326 
I have fixed the autoupdate.
Please hellp me review it again.
Thanks!

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Pull request merged**